dialog-polyfill.js

```diff
@@ -198,7 +198,9 @@
         }).join(', ');
         target = this.dialog_.querySelector(query);
       }
-      document.activeElement && document.activeElement.blur && document.activeElement.blur();
+      if (document.activeElement && document.activeElement.blur && document.activeElement.nodeName.toLowerCase() !== "body") {
```

Comment:
I'm confused, we're not explicitly calling `blur` here or below. So the 2nd bug in the related document shouldn't be triggered. What am I missing?